get_subs_and_view_count_from_api.py

```diff
@@ -0,0 +1,72 @@
+import urllib.request
+import json
+import pandas as pd
+
+
+"""
+This code is to concat all the files and get the subs, views and video count from each 
+of the channels from which we collected the data from. Run this once if a new channel/s is 
+added.
+date: 18-05-2019
+"""
+
+
+key = "AIzaSyAytjsDAjl1IpXXXpa_1i8JKImxralGy6Y" # This the google api key
```

Comment:
Always a good idea to have the API access key outside the code. Maybe have it in a config file and add the file to gitignore. 
Since it is open source, anybody can access the key

Reply:
Delete this comment?